docs/knowledge_base/tips/dataiku/index.md

```diff
@@ -0,0 +1,255 @@
+                                          'ActiveId': active_id,
+                                          'MetricType': "AUC",
+                                          'Value': value})
+        except Exception as e:
+            print(e)
+            pass
+
+metrics_df = pd.DataFrame(metrics_table)
+```
+
+## Preparing Data for Visualization
+
+Before visualization, you may need to further prepare your data. For instance, you can concatenate certain columns to form a unique identifier or name for each entry. Here, we add a 'Name' column to our DataFrame.
+
+```python
+metrics_df['Name'] = metrics_df[['Project', 'ModelId', 'MetricType']].agg(' - '.join, axis=1)
+```
+
+## Creating and Running a Taipy Application
+
+Define the GUI using Taipy's markdown-like syntax. This GUI will include dynamic selectors for projects and metrics, a table to display data, and a bar chart for visualizing metric values.
+
+```python
+from taipy.gui import Gui
+
+selected_projects = list(metrics_df['Project'].unique())
+selected_metrics = ["AUC"]  # Initially focusing on AUC metrics.
+
+def show_projects(metrics_df, selected_projects, selected_metrics):
+    return metrics_df[(metrics_df['Project'].isin(selected_projects)) &
+                      (metrics_df['MetricType'].isin(selected_metrics))]
+
+md = """
+# Dataiku **Overview**{: .color-primary}
+
+**Projects**
+
+<|{selected_projects}|selector|lov={list(metrics_df['Project'].unique())}|dropdown|multiple|class_name=fullwidth|label=Projects|>
+
+**Metrics**
+
+<|{selected_metrics}|selector|lov={list(metrics_df['MetricType'].unique())}|dropdown|multiple|class_name=fullwidth|label=Metrics|>
+
+<|{show_projects(metrics_df, selected_projects, selected_metrics)}|table|filter|>
+
+<|{show_projects(metrics_df, selected_projects, selected_metrics)}|chart|x=Name|y=Value|type=bar|>
+"""
+
+Gui(md).run()
+```
+
+
+
+# Dataiku Projects with Taipy Scenarios
+
+## Scenarios and Dataiku Integration
+
+Creating and executing projects on Dataiku involves several steps, from setting up your Dataiku instance to defining and running projects. Here's a step-by-step guide on how to seamlessly integrate Dataiku projects with Taipy scenarios:
+
+**Requirements:**
+
+- A Dataiku Instance.
+
+**1 - Create a Dataiku Project**
+
+- **Navigate to Dataiku DSS:** Access the dashboard where you want to create the project.
+
+- **Create a Project:** Click on the "New Project" button and select "Create."
+
+- **Define Project Details:** Enter a name for your project and choose the type of project you wish to create (e.g., Visual Analysis, Data Preparation).
+
+**2 - Define Dataiku Project Logic**
+
+- **Prepare the Data:** Create datasets, recipes, and models required for your project in the Flow section.
+
+- **Write Code:** Write code for data processing, analysis, or any other tasks within notebooks or custom recipes.
+
+Here's an example of how parameters can be used in a Dataiku project to control the workflow and obtain results:
```

Comment:
In the article, I would assume these steps (1 and 2) as a prerequisite. It is already done in Dataiku.
We don't want to document how to create a Dataiku project. This is not our goal plus it's going to be painful to maintain.